src/core/messenger/scroll.ts

```diff
@@ -91,7 +90,7 @@ const addScrollListener = (
 	respond: Respond,
 ): void => {
 	if (iframeCounter === 0) {
-		addEventListener(w, 'scroll', onScroll, {
+		w.addEventListener('scroll', onScroll, {
```

Comment:
While we're here could we replace `w` with a direct reference to `window` in this module? It's a little surprising to see it aliased.